spec/lib/facilities/vha_facility_spec.rb

```diff
@@ -1,89 +1,171 @@
+          expect(facility.address['mailing']).to eq({})
         end
+      end
 
-        it 'should parse mailing address correctly' do
-          VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
-            expect(facility.address['mailing']).to eq({})
-          end
+      it 'should parse mailing address correctly' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility.address['physical']).to eq('address_1' => '1501 Roxas Boulevard',
+                                                     'address_2' => 'NOX3 Seafront Compound',
+                                                     'address_3' => nil, 'city' => 'Pasay City',
+                                                     'state' => 'PH', 'zip' => '01302')
         end
+      end
 
-        it 'should parse mailing address correctly' do
-          VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
-            expect(facility.address['physical']).to eq('address_1' => '1501 Roxas Boulevard',
-                                                       'address_2' => 'NOX3 Seafront Compound',
-                                                       'address_3' => nil, 'city' => 'Pasay City',
-                                                       'state' => 'PI', 'zip' => '01302')
-          end
+      it 'should include just be 5 digit if zip +4 is empty' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility.address['physical']['zip']).to eq('01302')
+        end
+      end
+
+      it 'should include zip +4 when available' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility_2.address['physical']['zip']).to eq('04330-6796')
         end
+      end
+
+      it 'should include websites for facilities' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility_2.website).to eq('http://www.maine.va.gov/')
+        end
+      end
 
-        it 'should include just be 5 digit if zip +4 is empty' do
+      it 'should include active status for facilities' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility_2.active_status).to eq('A')
+        end
+      end
+
+      it 'should indicate if a facility is mobile' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility.mobile).to eq(false)
+          expect(facility_2.mobile).to eq(true)
+        end
+      end
+
+      it 'should get the correct classification name' do
+        VCR.use_cassette('facilities/va/vha_facilities_limit_results') do
+          expect(facility.classification).to eq('Other Outpatient Services (OOS)')
+          expect(facility_2.classification).to eq('VA Medical Center (VAMC)')
+        end
+      end
+
+      context 'mental health data' do
```

Comment:
When describing a context, start its description with "when" or "with".

```suggestion
  context 'with mental health data' do
```